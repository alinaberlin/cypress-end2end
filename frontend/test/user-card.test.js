import { mount } from '@vue/test-utils'
import UserCard from '@/components/user-card.vue'

describe('UserCard component', () => {
  test('Renders an example user', () => {
    const wrapper = mount(UserCard, {
      propsData: {
        user: {
          name: 'Armagan Amcalar',
          age: '36',
          photos: [{ filename: '/test.jpg', likedBy: [{ name: 'Mihri' }] }],
        },
      },
    })
    expect(wrapper).toMatchSnapshot()
  })
})
