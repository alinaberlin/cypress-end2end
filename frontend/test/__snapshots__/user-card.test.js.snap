// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserCard component Renders an example user 1`] = `
<div class="box">
  <h1>Armagan Amcalar (36)</h1>
  <p>Bio: This user has no bios.</p>
  <h2>Photos (1)</h2>
  <div class="photo"><img src="https://picsum.photos/test.jpg">
    <p>Mihri</p>
  </div>
</div>
`;
